roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -65,16 +69,14 @@ def run(self, tmp, task_vars):
 
     def required_images(self, task_vars):
         deployment_type = get_var(task_vars, "deployment_type")
-        # FIXME(juanvallejo): we should handle gracefully with a proper error
-        # message when given an unexpected value for `deployment_type`.
-        image_base_name = self.docker_image_base[deployment_type]
+        if deployment_type not in self.docker_image_base:
+            msg = "Unable to determine required images for deployment type {deployment}"
+            raise OpenShiftCheckException(msg.format(deployment=deployment_type))
```

Comment:
@juanvallejo @sosiouxme I'm thinking here, would it be best to fail a check when we meet an unexpected deployment_type, or should we simply skip the check (by implementing `is_active`)?

In your opinion, what would be the best user experience?